src/App.java

```diff
@@ -1,5 +1,59 @@
 public class App {
+    public static void printBoard(String[][] board) {
+        for (int i = 0; i < 8; i++) {
+            for (int k = 0; k < 4 || (k == 4 && i == 7); k++) {
+                for (int j = 0; j < 8; j++) {
+                    for (int l = 0; l < 4 || (l == 4 && j == 7); l++) {
+                        if (j == 0 && l == 0) {
+                            if (k == 2) {
+                                System.out.print(8 - i + " ");
+                            } else {
+                                System.out.print("  ");
+                            }
+                        }
+                        if (k == 0 || l == 0 || k == 4 || l == 4) {
+                            System.out.print("# ");
+                        } else if (k == 2 && l == 2 && board[i][j] != null) {
+                            System.out.print(board[i][j] + " ");
+                        } else {
+                            System.out.print("  ");
+                        }
```

Comment:
Write this method in its own class Board. and the name of the methods should be printPlayer1Board. Because this is just for the player1 perspective we also need to print board for player2 perspective as well.